### PR TITLE
[Merged by Bors] - Add `ListOrganizations` operation

### PR DESCRIPTION
**Pull request by sheldonvaughn**

### CODE REVIEW

File: src/organizations/operations/list_organizations.rs [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+#[derive(Debug, Serialize)]
+pub struct DomainFilters<'a>(UrlEncodableVec<&'a str>);
+
+impl<'a> From<Vec<&'a str>> for DomainFilters<'a> {
+    fn from(domains: Vec<&'a str>) -> Self {
+        Self(domains.into())
+    }
+}
+
+#[derive(Debug, Serialize)]
```
Comment (maxdeviant) [inline src/organizations/operations/list_organizations.rs]: Could we move this `derive` down so it's below the comment and right above the `pub struct` line?

File: src/organizations/operations/list_organizations.rs [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                      "allow_profiles_outside_organization": false,
+                      "created_at": "2021-06-25T19:07:33.155Z",
+                      "updated_at": "2021-06-25T19:07:33.155Z",
+                      "domains": [
+                        {
+                          "domain": "foo-corp.com",
+                          "id": "org_domain_01EHZNVPK2QXHMVWCEDQEKY69A",
+                          "object": "organization_domain"
+                        },
+                        {
```
Comment (maxdeviant) [inline src/organizations/operations/list_organizations.rs]: Could we remove this second organization domain just so the test looks a bit more realistic?

**Review by maxdeviant — APPROVED** [review]
Just a few minor tweaks 👍🏻

**Comment by sheldonvaughn** [comment]
bors merge

**Pull request closed without merging**